spark/dl/src/main/scala/com/intel/analytics/bigdl/dataset/image/LocalGreyImgReaderWithName.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.dataset.image
+
+import java.awt.color.ColorSpace
+
+import com.intel.analytics.bigdl.dataset.Transformer
+
+import scala.collection.Iterator
+
+/**
+ * Read BGR images from given paths,
+ * return LabeledBGRImage and the file name.
+ */
+object LocalGreyImgReaderWithName {
+  Class.forName("javax.imageio.ImageIO")
```

Comment:
Can you make the name as one variable in LabeledGreyImage, otherwise seems there are many duplicate codes